### PR TITLE
Cloudinary updates

### PR DESCRIPTION
**Pull request by aleima11**
[card](https://trello.com/c/fSH5wvm1/2-cloudinary-on-ag)
- The URL being constructed here no longer works
- Passing in the URL now instead

### CODE REVIEW

File: packages/react-ui-ag/src/Listings/Listing.js [hunk 1]
```diff
@@ -31,7 +31,12 @@ export default class Listing extends Component {
     theme: PropTypes.object,
     className: PropTypes.string,
     onClick: PropTypes.func,
-    photos: PropTypes.object,
+    photos: PropTypes.arrayOf(
```
Comment (tadjohnston) [inline packages/react-ui-ag/src/Listings/Listing.js]: This isn't what you think it is.. https://github.com/rentpath/react-ui/pull/365/files#diff-be83dd49bf57212006170863d5504aa5L86

We should probably just rename this prop to `photoConfig` or something else.

Comment (aleima11) [reply packages/react-ui-ag/src/Listings/Listing.js]: yeah makes more sense. Doesn't make sense anymore to construct the url in react-ui.

Comment (aleima11) [reply packages/react-ui-ag/src/Listings/Listing.js]: And I went redo crazy before I realized. This is super WIP.

**Comment by tadjohnston** [comment]
That should be the responsibility of the listing then and none of it should be done here.  The reason that configuration was done was to allow for multiple servers (load balancers) to be used and multiple dimensions for srcset usage. It was probably overkill to ever have react-ui even worry about it in the first place.

**Comment by aleima11** [comment]
So is there an issue with me removing that logic from react-ui and just having the photo url passed in?

**Comment by tadjohnston** [comment]
Nope, no issue, but then we can remove the `photo` prop from Listing https://github.com/rentpath/react-ui/pull/365/files#diff-cbc7d33b278cde12ded8005102f3bf1eR34.

**Comment by aleima11** [comment]
yeah for sure, those changes are all wrong.

**Comment by aleima11** [comment]
I left the server and then path, which will be the whole second part of the url. This way you can still do srcset. Just have to build in dimensions to the path before hand.

File: stories/ag/Listings/DesktopListing.js [hunk 2]
```diff
@@ -15,7 +15,6 @@ const props = {
   },
   photos: {
     server: 'https://image.rent.com/',
-    dimensions: '280-120',
```
Comment (nray621) [inline stories/ag/Listings/DesktopListing.js]: Sorry, I know this is a nitpick and a bunch of stories are currently broken anyway, but would you mind adding the dimensions to the URLs in `stories/ag/Listings/baseListing.js` so the story still has the proper dimensions?

File: packages/react-ui-core/package.json [hunk 3]
```diff
@@ -53,6 +53,7 @@
     "classnames": "^2.2.6",
     "form-serialize": "0.7.2",
     "prop-types": "^15.6.1",
+    "react": "^16.4.1",
```
[the remark this answers is not in the document]
Comment (aleima11) [reply packages/react-ui-core/package.json]: oopsies. Will do

**Comment by tadjohnston** [comment]
Please make sure you use `git cz` format on your commit before merge.

**Pull request merged**